src/main/java/io/stargate/sgv2/jsonapi/grpc/retries/impl/JsonApiGrpcRetryPolicy.java

```diff
@@ -56,7 +58,10 @@ private boolean isValidServerSideTimeout(Metadata trailers) {
     // if we have trailers
     if (null != trailers) {
       // read, write and CAS write timeouts will include one of two trailers
-      return trailers.containsKey(READ_TIMEOUT_KEY) || trailers.containsKey(WRITE_TIMEOUT_KEY);
+      if (trailers.containsKey(READ_TIMEOUT_KEY) || trailers.containsKey(WRITE_TIMEOUT_KEY)) {
+        logger.warn("DEADLINE_EXCEEDED with read/write timeout trailers, retrying");
```

Comment:
This could be bit noisy as it has very little information on which request, by which client, for what operation etc it is for. Or does tagging add such metadata?
It should perhaps also included whether this is for reads or writes.

I guess we can add more information as needed based on how this works in production.